ui/src/table.tsx

```diff
@@ -943,16 +943,22 @@ export const
     }), [filteredItems, groups, expandedRefs, isMultiple, items, m, onFilterChange, selectedFilters, selection, sort, setFiltersInBulk])
 
     return (
-      <div data-test={m.name} style={{ position: 'relative', height: computeHeight() }}>
-        <Fluent.Stack horizontal horizontalAlign='space-between' verticalAlign='end'>
-          {groupable && !m.pagination && <Fluent.Dropdown data-test='groupby' label='Group by' selectedKey={groupByKey} onChange={onGroupByChange} options={groupByOptions} styles={{ root: { width: 300 } }} />}
-          {!!searchableKeys.length && <Fluent.SearchBox data-test='search' placeholder='Search' onChange={onSearchChange} value={searchStr} styles={{ root: { width: '50%', maxWidth: 500 } }} />}
-        </Fluent.Stack >
+      <div data-test={m.name} style={{ position: 'relative', height: computeHeight()  }}>
+        <Fluent.Stack horizontal horizontalAlign='space-between' verticalAlign="end"  >
+           <Fluent.Stack.Item align='start' styles={{ root: { width: '50%', maxWidth: 500, margin: "28px 0 0 0"} }}>
```

Comment:
I guess the `align` should be `end` here instead of `start` since we use horizontal Stack and according to the Fluent align docs:

_Defines how to align the StackItem along the x-axis (for vertical Stacks) or the y-axis (for horizontal Stacks)._

Also, what is the extra margin for?

```suggestion
           <Fluent.Stack.Item align='end' styles={{ root: { width: '50%', maxWidth: 500} }}>
```

Reply:
I had some issues aligning the two buttons and ended up adding some padding. This in turn pushed the search bar into the table. I tested increasing the whitespace at the top of the table. I think the main issue was either the vertical align or the start align.
![image](https://user-images.githubusercontent.com/44904160/203405653-0f617f4d-4a87-4cac-872d-33a2d2e47efb.png)

Hope this is ok now.